proxy/healthy_endpoints.go

```diff
@@ -22,7 +22,7 @@ func (h *healthyEndpoints) filterHealthyEndpoints(ctx *context, endpoints []rout
 	filtered := make([]routing.LBEndpoint, 0, len(endpoints))
 	for _, e := range endpoints {
 		dropProbability := e.Metrics.HealthCheckDropProbability()
-		if p < dropProbability {
+		if dropProbability > 0.05 && p < dropProbability {
```

Comment:
Hm, this does not look right as it not only influences logging but a logic of dropping the endpoint.

We can enhance logging to log once a second/minute and also keep drop counter in the e.Metrics for logging but maybe we should just remove logs and rely on the `passive-health-check.endpoints.dropped`.

Reply:
And when probability goes up this will flood logs anyway.

Reply:
The min probability cutoff (if we want it) should be implemented in updateStats()

Reply:
As long as we have no metrics that are good enough we let the logs.
The idea was that we cut the min. There was a lot of logs that were way too close to 0 to make sense. If you have all endpoints having a low error rate you just cycle through all of them, likely something we also want to address.